templates/unattended-upgrades.erb

```diff
@@ -52,6 +52,12 @@ Unattended-Upgrade::MailOnlyOnError "<%= @_mail['only_on_error'].to_s %>";
 <%- end -%>
 <%- end -%>
 
+<%- unless @sender.nil? -%>
+// Use the specified value in the "From" field of outgoing mails.
+// Defaults to "root"
+Unattended-Upgrade::Sender "<%= @sender %>";
+<%- end -%>
+
```

Comment:
I'd move this empty line to inside the block so there is no diff between the old and new version.